cmd/appliance/upgrade/cancel.go

```diff
@@ -139,65 +144,56 @@ func upgradeCancelRun(cmd *cobra.Command, args []string, opts *upgradeCancelOpti
 				appliancepkg.UpgradeStatusReady,
 				appliancepkg.UpgradeStatusFailed,
 			}
-			// wg is the wait group for the progressbars
-			wg           sync.WaitGroup
-			errorChannel = make(chan error, count)
 		)
-		cancelProgressBars := mpb.New(mpb.WithOutput(spinnerOut), mpb.WithWaitGroup(&wg))
```

Comment:
i don't understand why wg is removed, wouldn't it be better to resolve the underlying issue instead? we only loose control.

Reply:
I have been testing with wait groups a lot, and it never produces the result we want. Either the spinners get completed too early or not at all.